### PR TITLE
Feat/live 4085 hedera bot improvements

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

- Add tests for hedera specs
- Add test destination

### ❓ Context

- **Impacted projects**: `LLC`
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-4085`

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/families/hedera/specs.ts [hunk 1]
```diff
@@ -86,7 +85,7 @@ const hedera: AppSpec<Transaction> = {
     },
     {
       name: "Send max",
-      maxRun: 2,
+      maxRun: 1,
```
Comment (henrily-ledger) [inline libs/ledger-live-common/src/families/hedera/specs.ts:88]: Is 2 run is buggy ?

**Review by henrily-ledger — APPROVED** [review]

File: libs/ledger-live-common/src/families/hedera/specs.ts [hunk 2]
```diff
@@ -95,14 +94,20 @@ const hedera: AppSpec<Transaction> = {
...
 
         return {
           transaction,
           updates: [{ recipient }, { useAllAmount: true }],
         };
       },
+      test: ({ account, accountBeforeTransaction, operation }) => {
+        botTest("Account balance should have decreased", () => {
+          expect(account.balance.toNumber()).toBeLessThanOrEqual(
+            accountBeforeTransaction.balance.minus(operation.value).toNumber()
```
Comment (hzheng-ledger) [inline libs/ledger-live-common/src/families/hedera/specs.ts:107]: why not
```expect(account.balance.toNumber()).Equal(
            accountBeforeTransaction.balance.minus(operation.value).toNumber()```

**Review by hzheng-ledger — COMMENTED** [review]
See our comments plz. Thanks

**Comment by hzheng-ledger** [comment]
> See our comments plz. Thanks

@lvndry Also, it seems that there are still some errors from the last run of Hedera error with this PR. Are these errors supposed to be fixed in this PR?

**Review by hzheng-ledger — APPROVED** [review]
looks good to me

**Pull request merged**